src/challenges/challengeOne/3.Table/Table.test.tsx

```diff
@@ -1,8 +1,59 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Table } from "./Table";
-
-// describe("Table", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { render, screen, within } from "@testing-library/react";
+import { Table, TableProps } from "./Table";
+
+const TableHeader = [
+  { property: "name", label: "Name" },
+  { property: "age", label: "Age" },
+  { property: "email", label: "Email" },
+];
+
+const TableRows = [
+  { id: 1, name: "John Doe", age: 30, email: "johndoe@example.com" },
+  { id: 2, name: "Jane Smith", age: 25, email: "janesmith@example.com" },
+];
+describe("Table", () => {
+  const renderComponent = (additionalProps?: any) => {
+    const defaultProps: Partial<TableProps> = {
+      columns: TableHeader,
+      data: TableRows,
+    };
+    render(<Table {...defaultProps} {...additionalProps} />);
+  };
+
+  it("renders without error", () => {
+    renderComponent();
+    expect(screen).not.toBeNull();
+  });
+
+  it("renders table headers correctly", () => {
+    renderComponent();
+    const tableHeaders = screen.getAllByRole("columnheader");
+    expect(tableHeaders).toHaveLength(3);
+    expect(tableHeaders[0]).toHaveTextContent("Name");
+    expect(tableHeaders[1]).toHaveTextContent("Age");
+    expect(tableHeaders[2]).toHaveTextContent("Email");
```

Comment:
Would be far better to just do 

`expect(screen.getByText("Name")).toBeInTheDocument();`

Reply:
This would be applied to all your tests.